src/JSHandle.re

```diff
@@ -0,0 +1,16 @@
+type t;
+
+/* Should be Js.null(ElementHandle.t) but these aren't currently distinct types.
+ * I think this is ok? */
+[@bs.send] external asElement : t => Js.null(t) = "";
+
+[@bs.send] external dispose : t => Js.Promise.t(unit) = "";
+
+/* TODO: executionContext */
+
+[@bs.send] external getProperties : t => Js.Promise.t(Js.Dict.t(t)) = "";
```

Comment:
This returns an ES6 Map. This definitely needs a test just to see if treating it as a dict works is possible.